tensorflow/lite/tools/optimize/operator_property.cc

```diff
@@ -372,6 +372,11 @@ OperatorProperty GetOperatorProperty(const ModelT* model, int subgraph_index,
       property.outputs = {{0, {}}};
       property.restrict_same_input_output_scale = true;
       property.version = 2;
+    case BuiltinOperator_UNPACK:
+      property.inputs = {{0, {}}};
+      property.arbitrary_outputs = true;
```

Comment:
This is outside your PR, but could you update BuiltinOperator_SPLIT to follow
your ordering of inputs and arbitrary_outputs? I think what you did is better.

Thanks!